cstring.h

```diff
@@ -1,944 +1,945 @@
-#ifndef CSTRING_LIB
```

Comment:
This is wrong! The entire file has been deleted and added back, some kind of error. It is necessary that only the really modified parts of the code are highlighted. Please fix it